Controllers/PrayerRequestsController.cs

```diff
@@ -0,0 +1,81 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using WebApi.Models;
+using WebApi.Data;
+namespace WebApi.Controllers
+{
+    [Route("api/[Controller]")]
+    [ApiController]
+    public class PrayerRequestsController : ControllerBase
+    {
+        private readonly ApiContext _context;
+
+        public PrayerRequestsController(ApiContext context)
+        {
+           _context = context;
+        }
+      
+      //Create and Edit prayer request
+      //POST: api/newRequest
+      [HttpPost("newAsk")] //can this just be "new" or does it need a new name because "new" is already being used in PrayerRequest
+      public JsonResult CreatePrayerRequest(PrayerRequest PrayerRequest)
```

Comment:
This isn't going to work because what you are passing into your function is currently saying you are passing in two unnamed variables of type PrayerRequest.   In your function public JsonResult CreatePrayerRequest() you the second prayerRequest should have the first letter lowercase to let the compiler know that you are naming a variable and not calling the PrayerRequest type twice.  So it should be JsonResult CreatePrayerRequest(PrayerRequest prayerRequest)  and then everywhere you are calling on that variable you need to change it to a lowercase p so that you are calling the variable and not the data type.  Let me know if this doesn't make sense.